### ISSUE TITLE
Duplicate class com.twilio.audioswitch.* when using this SDK in a project that depends on com.twilio.audioswitch

### ISSUE THREAD

**Issue by bennnjamin** [livekit/client-sdk-android#571]
**Describe the bug**
I have another dependency in my project that depends on com.twilio.audioswitch. When trying to add this SDK as a project dependency I cannot build the project due to duplicate class errors

**To Reproduce**
1. Add client-sdk-android as a dependency
2. Add com.twilio.audioswitch as a dependency (or any library that includes com.twilio.audioswitch as a dependency)

**Expected behavior**
Project should compile when using this dependency or documentation should explain this behavior when using projects that also have com.twilio.audioswitch as a dependency, similar to the LiveKit fork of WebRTC.

**Comment by wellingtonrib** [comment]
Some for me. Do you guys have any workaround for that?

**Comment by bennnjamin** [comment]
@wellingtonrib Yes you need to remove twilio's audioswitch gradle dependency and replace it with the exact version that LiveKit uses (which is a public fork)
